<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardDialog</class>
 <widget class="QDialog" name="KeyboardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="capsLayout">
       <item>
        <widget class="QPushButton" name="pbCaps">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Caps
Lock
</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbShift">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Shift</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="buttonsLayout"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="enterLayout">
       <item>
        <widget class="QPushButton" name="pbBackspace">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;--</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbEnter">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;-'</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
